pkg/dashboard/dashboard.go

```diff
@@ -78,3 +115,54 @@ func Dashboard(opts Options) http.Handler {
 		writeTemplate(tmpl, opts, &data, w)
 	})
 }
+
+func calculateContainerCost(costPerCPUFloat float64, costPerGBFloat float64, c summary.ContainerSummary) float64 {
+	cpuCost := costPerCPUFloat * (c.Requests.Cpu().AsApproximateFloat64() + c.Limits.Cpu().AsApproximateFloat64()) / 2
+	memCost := costPerGBFloat * (ConvertToGB(c.Requests.Memory().Value()) + ConvertToGB(c.Limits.Memory().Value())) / 2
+	return toFixed(cpuCost+memCost, 4)
```

Comment:
If `Limits` are not set, we should only use `Requests`

Reply:
I updated the code. Thank you